.github/workflows/cleanup.yml

```
@@ -79,17 +81,13 @@ jobs:
       - name: Delete PostgreSQL namespace schema
         if: github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch'
         run: |
-          # Install PostgreSQL client
-          sudo apt-get update
-          sudo apt-get install -y postgresql-client
-
           # Get list of all databases except system databases
-          DATABASES=$(psql '${{ secrets.DATABASE_INSTANCE_URL }}' -At -c "SELECT datname FROM pg_database WHERE datname NOT IN ('postgres', 'template0', 'template1')")
+          DATABASES=$(kubectl exec -n postgresql postgresql-0 -- psql -U postgres -At -c "SELECT datname FROM pg_database WHERE datname NOT IN ('postgres', 'template0', 'template1')")
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)